httpspanneroptions.go

```diff
@@ -0,0 +1,74 @@
+package money
+
+import (
+	"fmt"
+	"net/http"
+)
+
+// Option types for a HTTPSpanner used to quickly change a HTTPSpanners state.
+type HTTPSpannerOptions func(*HTTPSpanner)
+
+// A list of a HTTPSpanners different states.
+type SubTracer func(*http.Request) (*HTTPTracker, error)
+type Starter func(*http.Request) (*Span, error)
+type Ender func(*http.Request) (*HTTPTracker, error)
+type Off func(*http.Request)
+
+// trackExtractor extracts a tracker from a request's context. Is used
```

Comment:
I don't understand this comment here.  What is trackExtractor?

Reply:
Updated and proof read with new commit.